app/models/user.rb

```diff
@@ -188,7 +188,7 @@ def unread_inbox_comments
     inbox_comments.find(:all, :conditions => {:read => false})
   end
   def unread_inbox_comments_count
-    inbox_comments.count(:all, :conditions => {:read => false})
+    inbox_comments.with_feedback_comment.count(:all, :conditions => {:read => false})
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.